helm_resource/README.md

```diff
@@ -47,6 +48,9 @@ reference.
 
 `release_name`: The name of the release. If not specified, defaults to the Tilt UI resource name.
 
+`namespace`: Install the release into the specified namespace. Also, when specified, the chart
+may only create objects in that particular namespace.
+
```

Comment:
Calling out an edge case seen when trying to set a namespace on a chart that produces objects in multiple namespaces. 

This doesn't work as the kubectl command will lead to errors like `the namespace from the provided object "abc" does not match the namespace "xyz". You must pass '--namespace abc' to perform this operation.`.